.github/workflows/sync-tracks.yml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
     inputs:
       tracks:
-        description: 'List of repos to rerun the syncing workflow for, formatted as JSON array, e.g. ["exercism/julia"]'
+        description: 'List of track repos to rerun the syncing workflow for, formatted as JSON array, e.g. ["exercism/julia"]'
```

Comment:
`workflow_dispatch` will take whatever repos you give it, it doesn't apply a filter.